util/reggen/fpv_csr.sv.tpl

```diff
@@ -191,7 +199,14 @@ module ${mod_base}_csr_assert_fpv import tlul_pkg::*;
       if (d2h.d_valid) begin
         if (pend_trans[d_source_idx].wr_pending == 1) begin
           if (!d2h.d_error && regwen[hro_idx]) begin
+            % if lblock == "clkmgr":
```

Comment:
Thanks so much for updating this: I would not have caught it, mostly because I can't run these tests.

Reply:
By the way, do we check regwens in FPV?

Reply:
The funny thing about this template is that it's used for generating `clkmgr_csr_assert_fpv.sv` which is also used during regular block-level DV (that's how I found out about the need for changing this file). Whether we check the regwens in FPV I don't know. TBH, I don't think we have an FPV setup for clkmgr, do we?

Reply:
In any case the test would need to be created, since there is no way to infer which CSR a regwen is meant to affect. No magic bullets for these besides careful review.